SonarQube.Common/ConsoleLogger.cs

```diff
@@ -17,6 +17,10 @@ public class ConsoleLogger : ILogger
     {
         private const LoggerVerbosity DefaultVerbosity = VerbosityCalculator.InitialLoggingVerbosity;
 
+        public const ConsoleColor DebugColor = ConsoleColor.DarkCyan;
+        public const ConsoleColor WarningColor = ConsoleColor.Yellow;
+        public const ConsoleColor ErrorColor = ConsoleColor.Red;
```

Comment:
I used the same warning and error colours as for standard MSBuild console output. I'm not sure if MSBuild colorised verbose/debug output differently; I'll check.